docs/connections/connector-types/mysql-connector.md

```diff
@@ -0,0 +1,31 @@
+---
+title: MySQL/MariaDB Connector
+---
+
+Macrometa collection connectors allow you to extract data from or send data to an existing MySQL or MariaDB collection.
```

Comment:
```suggestion
Macrometa collection connectors allow you to extract data from or send data to an existing MySQL or MariaDB table.
```
@diana-macrometa - Let's change `collection` to `table` since we are considering MySQL and MariaDB as RDBMS.